nodes/widgets/ui_chart.html

```diff
@@ -108,7 +123,28 @@
             },
             labelStyle: function () { return this.name ? 'node_label_italic' : '' },
             oneditprepare: function () {
-                const propertyType = { value: 'property', label: RED._('@flowfuse/node-red-dashboard/ui-chart:ui-chart.label.key') }
+                const node = this
+                // add "Clear History" button
+                const clearHistoryBtn = $('<button>')
+                    .attr('class', 'ui-button ui-corner-all ui-widget leftButton')
+                    .attr('id', 'node-dialog-clear-history')
+                    .text('Clear Chart History')
+                    .on('click', () => {
+                        console.log('clear history')
+                        // call the HTTP Admin Endpoint
+                        $.ajax({
+                            url: `/dashboard/chart/${node.id}/clear`,
```

Comment:
Absolute path will fail if user has set editor path

On FF, this can be set under settings -> editor -> Editor URL

Reply:
How do I access this from within the node though? I've assigned the `httpAdminRoot` locally on my machine, but `RED.settings.httpAdminRoot` isn't a thing?

Reply:
Ignore ^^ - just use a relative path 🤦